hello! this is under construction

## links
- [twitter](https://twitter.com/llabueg)
- [orcID](https://orcid.org/0000-0002-6879-3954)
- `linelle [dot] abueg [at] rockefeller [dot] edu`

## about
I am a bioinformatician working for the Vertebrate Genome Lab at Rockefeller University, where we aim to generate high-quality reference genomes across for all vertebrate species. We are currently in Phase 1 of our effort, wherein we are sequencing a taxonomically representative species for each vertebrate order. 

## presentations
- Brajuka, N., **Abueg, L.**, Fedrigo, O., Jarvis, E.D., Formenti, G., and HPRC-T2T Consortium. (2022). Validation of a near telomere-to-telomere genome assembly for HG002 with Hi-C and ultra-long reads.
  - presented by co-first-author N. Brajuka at the Human Pangenome Reference Consortium annual meeting
  - Bethesda, MD (Oct 11, 2022)
- **Abueg, L.**, Brajuka, N., Palmada-Flores, M., Gallardo, C., Grüning, B., Larivière, D., Nekrutenko, A., Ostrovsky, A., Schatz, M., Jarvis, E.D., Fedrigo, O., Formenti, G. (2022). [Reproducible and high-quality genome assembly and evaluation workflows in Galaxy](/posters/BoG2022/).
  - originally presented at the Biology of Genomes
  - Cold Spring Harbor Laboratory (May 11, 2022)

## publications
- Formenti, G., **Abueg, L.**, Brajuka, A., Brajuka, N., Gallardo, C., Giani, A., Fedrigo, O., Jarvis, E.D. (2022). Gfastats: conversion, evaluation and manipulation of genome sequences using assembly graphs. Bioinformatics. [html](https://doi.org/10.1093/bioinformatics/btac460) [pdf](https://academic.oup.com/bioinformatics/advance-article-pdf/doi/10.1093/bioinformatics/btac460/44550845/btac460.pdf)
- Uliano-Silva, M., Gabriel R. N. Ferreira, J., Krasheninnikova, K., Darwin Tree of Life Consortium, Formenti, G., **Abueg, L.**, Torrance, J., Myers, E. W., Durbin, R., Blaxter, M., McCarthy S.A. (2022, *preprint*). MitoHiFi: a python pipeline for mitochondrial genome assembly from PacBio High Fidelity reads. bioRxiv. [html](https://www.biorxiv.org/content/10.1101/2022.12.23.521667v1)
- Secomandi, S. & Gallo, G.R., [*et al.*, including **Abueg, L.**] (2022, *preprint*). Pangenomics provides insights into the role of synantropy in barn swallow evolution. bioRxiv. [html](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac460/6633308) [pdf](https://www.biorxiv.org/content/biorxiv/early/2022/03/29/2022.03.28.486082.full.pdf)
- Orrock, J.L., **Abueg, L.**, Gammie, S., and Munshi-South, J. (2021) Exome sequencing of deer mice on two California Channel Islands identifies potential adaptation to strongly contrasting ecological conditions. Molecular Ecology. [html](https://onlinelibrary.wiley.com/doi/full/10.1002/ece3.8357) [pdf](https://onlinelibrary.wiley.com/doi/epdf/10.1002/ece3.8357)
- **Abueg, L.** & Munshi-South, J. (2019) Landscape Genomics of White-Footed Mice (*Peromyscus leucopus*) along an Urban-to-Rural Gradient in the New York City Metropolitan Area. ProQuest (Master's thesis). [html](https://www.proquest.com/openview/ed9dcd2553ff19cf28c592157097f7be/1?pq-origsite=gscholar&cbl=18750&diss=y)

## education
- MS in Biological Sciences, Fordham Graduate School of Arts and Sciences (May 2019)
- BS in Biological Sciences, Fordham University (May 2018)
  - Minor in Philosophy, Minor in Classical Civilizations

## resources
tbd
